code/modules/projectiles/magazines/smgs.dm

```diff
@@ -260,3 +260,28 @@
 	w_class = SIZE_SMALL
 	max_rounds = 48
 	gun_type = /obj/item/weapon/gun/smg/nailgun
+
+//-------------------------------------------------------
+//P90, a classic SMG.
+
+/obj/item/ammo_magazine/smg/p90
+	name = "\improper FN P90 magazine (5.7×28mm)"
+	desc = "A 9mm magazine for the MP5."
+	default_ammo = /datum/ammo/bullet/smg
```

Comment:
Did you mean to use this ammo?

Reply:
Ah no, forgot to change it to the p90 type, thanks for letting me know.